### ISSUE TITLE
FOV QC analysis

### ISSUE THREAD

**Issue by joaolsf** [Nanostring-Biostats/CosMx-Analysis-Scratch-Space#160]
For the FOV QC analysis, it is not clear to me what the parameters max_prop_loss and max_totalcounts_loss. It seems they influence a lot the number of FOVs that are flagged assuming they define thresholds for the maximum loss in total counts in the grids in an fov compared to its neighbours. However it is not clear to me how they could be adjusted in a dataset-specific manner. Is there a guideline or rationale to define their values?

Thanks!

**Comment by mpsch09** [comment]
Hello, I had a similar question and was wondering if there was a follow-up to this question? Thanks!

**Comment by patrickjdanaher** [comment]
Better late than never: see the below excerpt from the function's (in-code) docs:

#' FOV QC 
#' 
#' Run the full FOV QC workflow: break each FOV into a 7x7 grid, compare grid squares 
#' to similar squares in other FOVs, and look for bardode bits whose genes are losing signal.
#' Recommended to use this function for one tissue/slide at a time, not across multiple tissues.
#' @param counts Raw counts matrix, cells in rows, genes in columns. Can be sparse or standard format.
#' @param xy 2-column matrix of cells' xy positions, aligned to rows of counts.
#' @param fov Vector of cells' FOV IDs, aligned to rows of counts.
#' @param tissue Vector giving cells' tissue membership. This argument ensures different tissues don't overlap in xy space. Default of NULL assumes all cells come from the same tissue.
#' @param barcodemap Data frame with two columns: "gene" and "barcode". Download the barcodemap for your panel
#' from https://github.com/Nanostring-Biostats/CosMx-Analysis-Scratch-Space/tree/Main/code/FOV%20QC.
#' @param **max_prop_loss** Maximum loss of efficiency allowed for any bit. E.g., a value of "0.3" means all FOVs with bias <log2(1 - 0.3) will be flagged.
#' @param **max_totalcounts_loss** Maximum loss of total expression allowed for any FOV. E.g., a value of "0.5" means all FOVs with total counts <50% of comparable spatial regions will be flagged.
#' @export
runFOVQC <- function(counts, xy, fov, tissue = NULL, barcodemap, max_prop_loss = 0.6, max_totalcounts_loss = 0.6) {
